index.js

```diff
@@ -107,7 +107,7 @@ function expand(options) {
 
     var val = utils.get(data, prop);
     if (dot) val += '.';
-    if (val) return val;
+    if (val || val == false) return val;
 
```

Comment:
> but actually I am a Java developer who just started to work with js.

no problem, you fixed a bug, it was a good change. 

I'll merge as soon as I have a chance, thank you